### PR TITLE
Feature/api 189

### PR DESCRIPTION
**Pull request by moniquejones08**

### CODE REVIEW

**Comment by moniquejones08** [comment]
@stuisme @BrandonEMiller @Tmidge @brianhyder

File: apiary.apib [hunk 1]
```diff
@@ -82,56 +85,60 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 
 + Response 200 (application/json)
-  + Attributes (Account)
+    + Attributes (Account)
 
 # Group User
+Check user endpoints.
 
-## User Collection [/accounts/{accountId}/users/{id}{?includeAudit}]
+## User Collection [/accounts/{accountId}/users{?includeAudit}]
```
Comment (larastocks) [inline apiary.apib]: Should this be a paged object? 
{?page,pagesize,orderby,includeAudit}

File: apiary.apib [hunk 2]
```diff
@@ -82,56 +85,60 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 
-### Get Users [GET]
 + Response 200 (application/json)
-  + Attributes (array[User])
+    + Attributes (User)
 
 # Group Account Products
+Check account products endpoints.
 
 ## Account Product Collection [/accounts/{accountId}/products{?q}]
```
Comment (larastocks) [inline apiary.apib]: Paged object - {?page,pagesize,orderby,includeAudit}

File: apiary.apib [hunk 3]
```diff
@@ -82,56 +85,60 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 + Parameters
-  + accountId: 1234567890 (number, required) - {INT} The Dude Platform account the user belongs too.
-  + id: 1234567890 (number, required) - {INT} The Dude Platform User Id that represents the user.
-  + includeAudit: true (boolean, optional) - Flag to return audit information from the api
+    + accountId: 1234567890 (number, required) - {INT} The Dude Platform account the user belongs too.
+    + includeAudit: true (boolean, optional) - Flag to return audit information from the api
 
 ### GET
-
 + Response 200 (application/json)
```
Comment (larastocks) [inline apiary.apib:117]: - Response 200 (application/json)
  - Attributes (Base Collection Response)
    - data (required, array[User])

**Comment by moniquejones08** [comment]
Fixed all of your comments @larastocks , thanks!!!

**Comment by larastocks** [comment]
+1

**Pull request merged**